open-company/open-company-web
# Whats new dot

#### NB: review after #849 

To test:
- open carrot
- wait a few seconds
- [x] do you NOT see a green dot besides your user avatar? Good
- add a new whats new item to headway
- refresh Carrot
- wait a few seconds
- [x] do you see a green dot besides the user menu? Good
- now open the menu
- [x] do you see a green dot besides what's new too? Good
- click Whats new
- [ ] do you see the headway widget in the right place? Good

NB: to make this work i removed the animation from the menu. Since that's almost always the first panel that opens i think it makes sense to remove all the panels animations at this point, WYT?

NB: this doesn't remove the page scroll when the menu (or probably another panel) appear, i won't fix it here though since that's fixed in #866
